examples/neuron/inferentia.yaml

```diff
@@ -0,0 +1,55 @@
+resources:
```

Comment:
Can we create another yaml that allows multiple accelerators, e.g,`accelerators: {A100:4, Inferentia:6}`? This way people can allow the system to choose whatever accelerators available.

The YAML can be brute force, i.e., do a `if..else` in setup and run.